recipes/tadarida-d/meta.yaml

```diff
@@ -3,12 +3,12 @@ package:
   version: "1.0"
 
 source:
-  fn: Tadarida-D-Tadarida-D_For-Galaxy-E.tar.gz
-  url: https://github.com/ThimotheeV/Tadarida-D/archive/Tadarida-D_For-Galaxy-E.tar.gz
-  md5: 8760d526641e3f22e237774eeee2be04
+  fn: Tadarida-D-master.tar.gz
+  url: https://github.com/ThimotheeV/Tadarida-D/archive/master.tar.gz
```

Comment:
@ThimotheeV please don't track the master branch, please always track releases or commits so that it is reproducible.
Furthermore, if you increase the version, you can reset the build-number to 0.